unikernel.ml

```diff
@@ -200,60 +255,75 @@ struct
         let password =
           json |> Yojson.Basic.Util.member "password" |> Yojson.Basic.to_string
         in
-        match validate_user_input ~name ~email ~password with
+        let form_csrf =
+          json |> Yojson.Basic.Util.member "form_csrf" |> Yojson.Basic.to_string
```

Comment:
The `member` may raise, we may want to protect that with a try .. catch.

But maybe we want another time to guard in a separate module all the Yojson usage, to me it looks like the yojson API is a bit hard to use if you don't like to catch exceptions.

Reply:
Let's leave this for a separate PR.

Reply:
see #66